index.bs

```diff
@@ -2512,8 +2519,14 @@ Methods</h4>
 
 	: <dfn>getChannelData(channel)</dfn>
 	::
+		<div class="correction" id="c2361-2">
+		  <span class="marker">Candidate Correction
+		  <a href="https://github.com/WebAudio/web-audio-api/issues/2361">Issue 2361-2.</a>
+		  </span>
+		  Use new Web IDL buffer primitives
+		</div>
 		According to the rules described in <a href="#acquire-the-content">acquire the content</a>
-		either [=get a reference to the buffer source|get a reference to=]
+		either <del cite=#c2361-2>get a reference to</del><ins cite=#c2361-2>[=ArrayBufferView/write=]</ins> into
```

Comment:
The phrasing for the new part is a bit confusing to me.  The new part says

> According to the rules described in acquire the content either write into...

But we're not writing anything at this point.  `getChannelData` allows you to get a copy or write at some later time; we're not writing any data now.

Is this the phrasing we really want to use?

@domenic WDYT?

Reply:
Yeah, I'm not sure I understood this part of the spec in its original formulation either; it didn't provide an algorithmic series of steps that explain what's going on. I'd be interested in seeing what the implementations do as an algorithm, so that we could make this spec precise instead.

Reply:
To be clear, the existing spec didn't really make sense either, as it was trying to apply the "get a reference to"/"get a copy to" algorithm to [[internal data]], which is not a BufferSource.

Reply:
Fair enough.  I'll try to rewrite it a little so it seems less awkward.